### ISSUE TITLE
Restart synchronizationJob least application privileged permissions returns a 401 Unauthorized for Application owner

### ISSUE THREAD

**Issue by dbollaer** [microsoftgraph/microsoft-graph-docs-contrib#8800]
I was testing out your api documentation.
I created an application with provisioning.
Then I created an application for resetting the provisioning application job.
I added the reset sync application as owner of the provisioning application.

When I give the reset sync application permission Synchronization.ReadWrite.All 
Everything works.
When I give my reset application permission Application.ReadWrite.OwnedBy
I keep getting a 401

```
{
  "error": {
    "code": "UnknownError",
    "message": "",
    "innerError": {
      "date": "2023-11-08T15:01:03",
      "request-id": "a68b1289-1105-45d8-9b13-03a60598f9c0",
      "client-request-id": "a68b1289-1105-45d8-9b13-03a60598f9c0"
    }
  }
}
```

Do I need to assign my application an Application Administrator role or Cloud Application Role?

Thank your clarifying this issue.

Why am I doing this?
The reason is that our vendor has some problems with their scim provisioning.
We noticed when we restart the job, everything comes back to normal.

Ideally we want to offer our reset script for their application.
But we do not want to give our vendor the permissions to reset all our sync jobs.

Thank your having a look.
--

<br class="Apple-interchange-newline">

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: deb447fb-b66f-53dc-e8aa-bebac66bc85a
* Version Independent ID: f190ff15-905c-212a-e3fa-e7b09f460227
* Content: [Restart synchronizationJob - Microsoft Graph v1.0](https://learn.microsoft.com/en-us/graph/api/synchronization-synchronizationjob-restart?view=graph-rest-1.0&tabs=http)
* Content Source: [api-reference/v1.0/api/synchronization-synchronizationjob-restart.md](https://github.com/microsoftgraph/microsoft-graph-docs-contrib/blob/main/api-reference/v1.0/api/synchronization-synchronizationjob-restart.md)
* Product: **applications**
* Technology: **microsoft-graph**
* GitHub Login: @ArvindHarinder1
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by FaithOmbongi** [comment]
Hi @ArvindHarinder1 - Can you help to investigate whether the Application.ReadWrite.OwnedBy app-only permission is sufficient for the [Restart synchronizationJob](https://learn.microsoft.com/en-us/graph/api/synchronization-synchronizationjob-restart?view=graph-rest-1.0&tabs=http) API? Thanks.

**Comment by FaithOmbongi** [comment]
Closing this issue in readiness for deactivation of GitHub issues. See this post for more information https://github.com/microsoftgraph/microsoft-graph-docs-contrib/issues/9146.

I've filed internal bug 2858108 that's assigned to @ArvindHarinder1 to help investigate and resolve the issue. Permissions information are defined in an internal permissions model so the fix can't be made through a manual docs PR in the docs repo.
